Dockerfile

```diff
@@ -12,4 +16,10 @@ ENV OPCUA_CERTIFICATE_DIR="/certificates"
 ENV OPCUA_OWN_CERTIFICATE_DIR="/certificates/pki/own"
 ENV OPCUA_CERTIFICATE_SUBJECT="CN=Opcua-extractor, C=NO, S=Oslo, O=Cognite, DC=docker.opcua"
 
+RUN mkdir -p /logs
+RUN chmod -R a+rw /logs
```

Comment:
You probably have to chmod `/certificates` as well.

Reply:
I checked, and it should have read access, do we need write access as well?

Reply:
Yup